critiquebrainz/data/dump_manager.py

```diff
@@ -269,10 +273,11 @@ def add_meta_files(tarfile, meta_files_dir):
     tarfile.add(os.path.join(meta_files_dir, 'SCHEMA_SEQUENCE'), arcname='SCHEMA_SEQUENCE')
 
 
-def create_base_archive(*, location, meta_files_dir=None):
+def create_base_archive(connection, *, location, meta_files_dir=None):
     """Creates a dump of all license-independent information: (users, license).
 
     Args:
+        connection (object): for supporting execution withing a transaction 
```

Comment:
What type of object? And there's a typo

Reply:
@paramsingh How should I write the type of object? Is there any sample which you can point out?

Reply:
I think it'll be `sqlalchemy.engine.Connection` or `sqlalchemy.engine.Transaction` here